src/Microsoft.AspNetCore.Routing.Abstractions/RouteValueDictionary.cs

```diff
@@ -606,9 +594,5 @@ private static void ValidatePropertyNames(Type type, PropertyHelper[] properties
                 }
             }
         }
-
-        private class PropertyCache : ConcurrentDictionary<Type, PropertyHelper[]>
```

Comment:
@rynowak I looked in history and you added this type. Was there a reason?

Reply:
Just because I like private classes